examples/package-not-found.js

```diff
@@ -0,0 +1,3 @@
+'use strict'
```

Comment:
I'll put this files inside a `test` folder :)

Reply:
All tests are inside examples (`incorrect-plugin.js` too)